paparazzi-junit4/src/test/java/app/cash/paparazzi/RenderExtensionTest.kt

```diff
@@ -1,3 +1,5 @@
+@file:Suppress("invisible_reference", "invisible_member")
```

Comment:
Consider using all uppercase to highlight that you're suppressing compiler errors

See https://github.com/JetBrains/kotlin/blob/master/compiler/frontend/src/org/jetbrains/kotlin/diagnostics/rendering/DefaultErrorMessages.java

```suggestion
@file:Suppress("INVISIBLE_REFERENCE", "INVISIBLE_MEMBER")
```